sapi/fpm/tests/status-basic.phpt

```diff
@@ -28,6 +28,7 @@ $expectedStatusData = [
     'max active processes' => 1,
     'max children reached' => 0,
     'slow requests'        => 0,
+    'memory peak'          => 2097152,
```

Comment:
I'm not sure if this is going to be stable so might be better to skip it so we don't have to update test later...